xblock/exceptions.py

```diff
@@ -21,7 +21,7 @@ class XBlockSaveError(Exception):
     """
     Raised to indicate an error in saving an XBlock
     """
-    def __init__(self, saved_fields, dirty_fields):
+    def __init__(self, saved_fields, dirty_fields, message=None, *args):
```

Comment:
let's leave out both, and if people want to extend it further, we can add that in a future PR